env.bzl

```
@@ -81,6 +106,10 @@ conda_create_rule = repository_rule(
             default = True,
             doc = "False if conda output should be shown",
         ),
+        "use_mamba": attr.bool(
+            default = False,
+            doc = "True if mamba should not be used",
```

I think it should be:

```suggestion
            doc = "True if mamba should be used",
```